### DISCUSSION TITLE
Corrupt Audio Files in \SOUNDS\en\SYSTEM for EdgeTX ver 2.6.0

### DISCUSSION THREAD

**Discussion by steveveach** [EdgeTX/edgetx#2306] [General]
Hello everyone. I'm a newbie and just got my RadioMaster TX16S Mark II Max transmitter today. It is loaded with EdgeTX ver. 2.6.0. The very first thing I did was try to copy the SD Card contents to a Windows folder for safe keeping. I ran into corrupt file errors that prevented Windows from copying some of the files (see attached).

Since I'm new, I don't want to open a bug ticket unnecessarily if this isn't a real issue, so I thought I would post here first.

1. Does anyone know if this is a reported issue? I couldn't find a ticket for it.
2. Does anyone know if these files are corrupted in earlier (or later) versions? I'm still not sure how to download different versions of these sound files.

Thanks.

![Corrupt](https://user-images.githubusercontent.com/48031888/189544860-2909e54a-00c6-4f61-8eea-a3089e6c4bb6.jpg)

**Comment by dlktdr** [comment]
They have shipped with some not so good SD cards. I would be purchase a good quality one to replace it with. 2.6 is an older version. I would also update it to 2.7.1 while your at it. Go to [buddy.edgetx.org](https://buddy.edgetx.org/) is an easy way. You can also use flasher to do it as well https://github.com/EdgeTX/flasher/releases. Both can also build up an SD card too.

You can find the sound files available here https://github.com/EdgeTX/edgetx-sdcard-sounds There are a few voices you can pick from as well.

The rest of the SD card content is available here https://github.com/EdgeTX/edgetx-sdcard/releases, scroll to the 2.7.0 section if you want to load the card up manually. Choose Assets, and pick the horus file.

**Reply by pfeerick** [reply]
Also, if you haven't tried this already, I would also suggest removing the microSD card from the radio, and using a SD card reader to verify - as sometimes downloads through the radio can also fail and give false errors, making you think the card is faulty. Be careful when you put it back in that you don't miss the slot though, and insert it into the depths of the radio, never to be found again, and forever rattle! 😆 And if you don't already do this, use the bootloader when you do modify the SD card, not when the main firmware is running - just ensures there is nothing active on the radio wanting to use the card.

**Comment by steveveach** [comment]
To restate the problem, the SD Card and all the other files on the SD Card are fine. It's just the files in my original post that are corrupted on the SD Card. That would seem to indicate that were installed corrupted at the factory at least for this version 2.6.0.

I will take your advice and see if I can find the same files somewhere online that are _not_ corrupted to replace those on my card that are.

Thanks.

**Comment by steveveach** [comment]
To close this thread, I did have to repair the SD card using the Windows 10 utility. Once I did this, there were no more read problems. I found good copies of the missing files from a previous version (I think release 2.5.0) and just copied them to my SD card which is using firmware version 2.6.0. That resolved the issue. Thanks.

**Reply by pfeerick** [reply]
Thanks for the update... glad you resolved it. If you're after the sound files in the future, you can always download them from the sound/voice pack repository ;) https://github.com/EdgeTX/edgetx-sdcard-sounds
